### ISSUE TITLE
[Documentation][InputComponent] The source code for the Input in error state is wrong

### ISSUE THREAD

**Issue by lucianmol** [vmware/clarity#2450]
**Select one ...** (check one with "x")

```
[x] bug
[ ] feature request
[ ] enhancement
```

### Expected behavior

The Angular Input error state code example should show the source for the input component above.

```
<form clrForm>
  <clr-input-container>
    <label>Full name</label>
    <input clrInput placeholder="Full name" name="name" [(ngModel)]="name" />
    <clr-control-helper>Helper text</clr-control-helper>
    <clr-control-error>This field is required!</clr-control-error>
  </clr-input-container>
</form>
```

### Actual behavior

It shows the source for the Textarea component.

### Reproduction of behavior

![screen shot 2018-07-11 at 16 00 27](https://user-images.githubusercontent.com/16926235/42573019-ae75bbb0-8523-11e8-884f-9da6a6c155e5.png)

### Environment details

* **Angular version:** 4.x.x

* **Clarity version:**

* **OS and version:**

* **Browser:** [all | Chrome XX | Firefox XX | IE XX | Safari XX | Mobile Chrome XX | Android X.X Web Browser | iOS XX Safari | iOS XX UIWebView | iOS XX WKWebView ]

Let me know if I can contribute with a PR!

**Comment by adityarb88** [comment]
@lucianmol: Thanks for reporting this issue. We will get this fixed soon.

**Comment by gnomeontherun** [comment]
Closed via #2507
